sys/psa_crypto/include/psa_ciphers.h

```diff
@@ -77,7 +78,9 @@ psa_status_t psa_cipher_cbc_aes_128_decrypt(const psa_key_attributes_t *attribut
                                             uint8_t *output,
                                             size_t output_size,
                                             size_t *output_length);
+#endif /* CONFIG_CIPHER_CBC_AES_128 */
```

Comment:
```suggestion
#endif /* MODULE_PSA_CIPHER_AES_128_CBC */
```

should match the #if above, same below